docs/asciidoc/context.adoc

```diff
@@ -576,7 +576,7 @@ a javadoc:Session[] but the lifecycle is shorter: *data is kept for only one req
   });
 
   post("/save", ctx -> {
-    ctx.flash("success", "Item created");          <1>
```

Comment:
just aligning the doc with what we have at the moment. There is no `ctx.flash(str,str)` method